app/models/account.rb

```diff
@@ -95,6 +95,22 @@ def reset!
     RedisEndpoint.reset!
   end
 
+  # Get admin emails associated with this account/site
+  def admin_emails
+    # Must run this against proper tenant database
+    Apartment::Tenant.switch(tenant) do
+      Site.instance.admin_emails
+    end
+  end
+
+  # Set admin emails associated with this account/site
+  def admin_emails=(emails)
+    # Must run this against proper tenant database
+    Apartment::Tenant.switch(tenant) do
+      Site.instance.admin_emails = emails
+    end
+  end
```

Comment:
Looking at the allowed params to `new`, it seems that the `admin_emails` is an array of emails, but this is not entirely clear that an array of emails is the input to this method when setting the values here or returning them in the getter.  (That is, there is no documentation on the method, as in yard docs with @param attributes.)  So, I'm left wondering whether we are working with arrays of strings or maybe CSV strings or what?